test/factories/volunteers.rb

```diff
@@ -72,8 +72,7 @@
 
     factory :volunteer_with_user do
       after(:create) do |volunteer|
-        volunteer.user = FactoryBot.create(:user_volunteer)
-        volunteer.save
+        volunteer.update(user: FactoryBot.create(:user_volunteer))
```

Comment:
You can just call `create` directly, no need for `FactoryBot.`

Reply:
I did it because I assumed create to be in the actual objects scope somehow.... very most likely you're right 😉

Reply:
We have `include FactoryBot::Syntax::Methods` in `test/test_helper.rb` so the methods are available everywhere in test cases :)